l3kernel/l3token.dtx

```diff
@@ -2827,7 +2822,7 @@ end
 %     \peek_meaning_remove:N,
 %   }
 %   The public functions themselves cannot be defined using
-%   \cs{prg_new_conditional:Npnn}.  Instead, the |TF|, |T|, |F| variants
+%   \cs{prg_new_conditional:Npn}.  Instead, the |TF|, |T|, |F| variants
```

Comment:
```suggestion
%   \cs{prg_new_protected_conditional:Npn}.  Instead, the |TF|, |T|, |F| variants
```
as `\peed_(catcode|charcode|meaning)_(remove):N(TF)` are all protected.

Reply:
I'll fix outside of the PR